shard-operator/src/main/java/com/redhat/service/bridge/shard/operator/BridgeExecutorServiceImpl.java

```diff
@@ -87,15 +88,6 @@ public Deployment fetchOrCreateBridgeExecutorDeployment(BridgeExecutor bridgeExe
 
         // TODO: All the Executor applications will push events to the same kafka cluster under the same kafka topic. This configuration will have to be specified by the manager for each Bridge instance: https://issues.redhat.com/browse/MGDOBR-123
```

Comment:
This TODO is wrong, I think the ticket is now MGDOBR-58

Reply:
@danielezonca actually I think this TODO has to be removed as the shard does not need anymore to know such details once this PR will be merged. 
Removed